src/core.jl

```diff
@@ -223,7 +223,9 @@ struct BoundaryFill{T<:Colorant} <: AbstractPolyFillAlgorithm
     end
 end
 
-BoundaryFill(;x::Int = 0, y::Int = 0, fill_value::Colorant = RGB(1), boundary_value::Colorant = RGB(1)) = BoundaryFill(x, y, fill_value, boundary_value)
+BoundaryFill(x::Int = 1, y::Int = 1; fill_value::Colorant = RGB(1), boundary_value::Colorant = RGB(1)) = BoundaryFill(x, y, fill_value, boundary_value)
+BoundaryFill(p::CartesianIndex{2}; fill_value::Colorant = RGB(1), boundary_value::Colorant = RGB(1)) = BoundaryFill(p[1], p[2], fill_value, boundary_value)
+BoundaryFill(p::Point; fill_value::Colorant = RGB(1), boundary_value::Colorant = RGB(1)) = BoundaryFill(p.y, p.x, fill_value, boundary_value)
```

Comment:
> The pixels to define the boundary using verts is filled with this colorant parameter. 
Without this parameter, the whole picture will be filled with the `fill_value`

The `boundary_value` is still `RGB(1)` if you pass `BoundaryFill(p; fill_value=Gray(1))`.

```suggestion
BoundaryFill(x::Int = 1, y::Int = 1; fill_value::Colorant = RGB(1), boundary_value::Colorant = fill_value) = BoundaryFill(x, y, fill_value, boundary_value)
BoundaryFill(p::CartesianIndex{2}; fill_value::Colorant = RGB(1), boundary_value::Colorant = fill_value) = BoundaryFill(p[1], p[2], fill_value, boundary_value)
BoundaryFill(p::Point; fill_value::Colorant = RGB(1), boundary_value::Colorant = fill_value) = BoundaryFill(p.y, p.x, fill_value, boundary_value)
```

Need to add a test to check this.